doc/user/radial_and_axial_expansion.rst

```diff
@@ -2,11 +2,7 @@
 Radial and Axial Expansion and Contraction
 ******************************************
 
-ARMI natively supports linear expansion in both the radial and axial dimensions.
-These expansion types function independently of one another and each have their
-own set of underlying assumptions and use-cases. The remainder of this section
-is described as follows: in Section :ref:`thermalExpansion` the methodology used
-for thermal expansion within ARMI is described.
+ARMI natively supports linear expansion in both the radial and axial dimensions. These expansion types function independently of one another and each have their own set of underlying assumptions and use-cases. The remainder of this section is described as follows: in Section :ref:`thermalExpansion` the methodology used for thermal expansion within ARMI is described.
```

Comment:
Do you mean to have this as one long line?

Reply:
Well, it's on purpose, but I can change it.

My _idea_ here is that it's probably better to let Sphinx decide where line wraps should happen, and let your IDE do the wrap, then to add a bunch of carriage returns in.

It's just an idea, I'm not married to it.

Reply:
Huh. Have we encountered issues with this? In many downstream projects we generally avoid the one long line approach in favor or breaking them up. I don't have a strong opinion, but would like consistency one way or another.

Reply:
Okay, I changed them over to shorter line lengths.